+p_xml: 'version="1.0" encoding="UTF-8" standalone="yes" '
+directive: DOCTYPE bugzilla SYSTEM "http://bugzilla.fieldtriptoolbox.org/page.cgi?id=bugzilla.dtd"
bugzilla:
  +@version: 4.4.1
  +@urlbase: http://bugzilla.fieldtriptoolbox.org/
  +@maintainer: r.oostenveld@donders.ru.nl
  bug:
    bug_id: "1742"
    creation_ts: 2012-09-25 17:32:00 +0200
    short_desc: problem with number of qsubcellfun output argument
    delta_ts: 2012-10-29 13:44:58 +0100
    reporter_accessible: "1"
    cclist_accessible: "1"
    classification_id: "1"
    classification: Unclassified
    product: FieldTrip
    component: qsub
    version: unspecified
    rep_platform: PC
    op_sys: Mac OS
    bug_status: CLOSED
    resolution: FIXED
    bug_file_loc:
    status_whiteboard:
    keywords:
    priority: P3
    bug_severity: normal
    target_milestone: '---'
    everconfirmed: "1"
    reporter:
      +content: r.oostenveld
      +@name: Robert Oostenveld
    assigned_to:
      +content: r.oostenveld
      +@name: Robert Oostenveld
    cc:
      - eelke.spaak
      - isheftel
      - marcel.zwiers
    comment_sort_order: oldest_to_newest
    long_desc:
      - +@isprivate: "0"
        commentid: "7595"
        comment_count: "0"
        who:
          +content: r.oostenveld
          +@name: Robert Oostenveld
        bug_when: 2012-09-25 17:32:54 +0200
        thetext: "Irina reported the following:\n\nIâ€™ve got an error when using the qsub:\n \n??? The left hand side is initialized and has an empty range of indices.\nHowever, the right hand side returned one or more results.\n \nError in ==> fexec at 153\n  [argout{:}] = feval(fname, argin{:});\n \nError in ==> qsubexec at 54\n  [argout, optout] = fexec(argin, optin);"
      - +@isprivate: "0"
        commentid: "7596"
        comment_count: "1"
        who:
          +content: r.oostenveld
          +@name: Robert Oostenveld
        bug_when: 2012-09-25 17:33:32 +0200
        thetext: "and she can reproduce it when simply calling for randn:\n \nNumargout at qsubcellfun line 202 is 0\n \nqsubcellfun(@randn, {2,2,2}, 'memreq', 1024, 'timreq', 60)\nstacking 3 matlab jobs in each qsub job\nsubmitting job xirisim_mentat286_p18528_b8_j001... qstat job id 785564.dccn-l014.dccn.nl\njob xirisim_mentat286_p18528_b8_j001 returned, it required NaN seconds and NaN GB\n%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%\n% an error was detected inside MATLAB, the diary output of the remote execution follows\n%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%\n??? The left hand side is initialized and has an empty range of indices.\nHowever, the right hand side returned one or more results.\n \nError in ==> fexec at 153\n  [argout{:}] = feval(fname, argin{:});\n \nError in ==> qsubexec at 54\n  [argout, optout] = fexec(argin, optin);"
      - +@isprivate: "0"
        commentid: "7599"
        comment_count: "2"
        who:
          +content: r.oostenveld
          +@name: Robert Oostenveld
        bug_when: 2012-09-25 17:49:59 +0200
        thetext: "I found the problem and fixed it\n\nroboos@mentat001> svn commit qsub/ test/test_bug1742.m \nSending        qsub/private/fexec.m\nAdding         test/test_bug1742.m\nTransmitting file data ..\nCommitted revision 6540."
      - +@isprivate: "0"
        commentid: "7871"
        comment_count: "3"
        who:
          +content: eelke.spaak
          +@name: Eelke Spaak
        bug_when: 2012-10-10 13:41:16 +0200
        thetext: |-
          The test script test_bug1742 was failing because the Torque cluster killed it. This happened because the child jobs that this test script submits of course were submitted at the end of the queue, thereby causing the test script itself to run for far too long.

          I now changed the script to specify the option backend=local to qsubcellfun. Since this also spawns a new process and also makes use of fexec, it should still test the same phenomenon. If I misunderstand this, just let me know :)
      - +@isprivate: "0"
        commentid: "8022"
        comment_count: "4"
        who:
          +content: r.oostenveld
          +@name: Robert Oostenveld
        bug_when: 2012-10-29 13:44:58 +0100
        thetext: changed the status of several bugs that were RESOLVED some time ago to CLOSED
